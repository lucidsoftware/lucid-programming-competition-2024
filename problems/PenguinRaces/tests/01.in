5
Pink - 15
Slider - 21
Macaroni - 45
Flippers - 24
Tux - 38