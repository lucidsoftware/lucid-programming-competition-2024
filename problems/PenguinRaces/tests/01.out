First: Pink
Second: Slider
Third: Flippers